### PR TITLE
Add Jabrank2's Islamabad file

### LINKED ISSUE #14: jabrank2's workflow for adding Islamabad to the Repository
## Islamabad, Pakistan
- [ ] Create an issue
- [ ] Create a branch
- [ ] Make changes
- [ ] Open a PR
- [ ] Collaborate
- [ ] Get approval to :ship: 
- [ ] Merge

### PR DESCRIPTION
**Pull request by jabrank2**
closes #14

### CODE REVIEW

File: jabrank2-Islamabad.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+,'NoisysEdgar'
+,'NoisysOrion'
+,'NoisysStage'
+,'OwnershipSTG'
```
Comment (jabrank2) [inline jabrank2-Islamabad.md]: I no like!

**Review by bionicle4365 — COMMENTED** [review]
I was hoping for good Islamabad restaurants.
I was disappointed. :-1:

File: jabrank2-Islamabad.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+,'NoisysEdgar'
+,'NoisysOrion'
+,'NoisysStage'
```
Comment (pmlevere) [inline jabrank2-Islamabad.md:3]: Very nice

**Comment by brianamarie** [comment]
Any good places to eat?

**Review by elexpander — APPROVED** [review]
good job

**Review by githubteacher — APPROVED** [review]

**Pull request closed without merging**